src/PantherTestCaseTrait.php

```diff
@@ -58,7 +58,7 @@
 
     public static function tearDownAfterClass()
     {
-        if (true === self::$stopServerOnTeardown) {
+        if (self::$stopServerOnTeardown) {
```

Comment:
Why did you make this change? IIRC it's always faster to explicitly compare values strictly instead of relying on the default PHP type-cast, WDYT?

Reply:
Well, `true === self::$stopServerOnTeardown` just produces another zval, which is then checked again with non-strict rules in the `if`. So if this property always contains a boolean already, you are only adding *more* work with the `true ===`

Reply:
If the gain is not significant as explained by @stof, would you be ok with this change ?

Reply:
I wasn't aware of this zval process making another check, TIL :) 
I'm ok :)